pkg/operator/v1alpha1helpers/helpers.go

```diff
@@ -8,14 +8,17 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes/scheme"
 
 	operatorsv1alpha1 "github.com/openshift/api/operator/v1alpha1"
+	installertypes "github.com/openshift/installer/pkg/types"
```

Comment:
you may not depend on this rep.

Reply:
Are you sure? It makes sense to avoid circular dependencies but it's already a dependency in [machine-config-operator](https://github.com/openshift/machine-config-operator/blob/master/pkg/operator/render.go#L11) and library-go doesn't seem to be imported anywhere in installer right now, is there another reason for not depending on installer? If so it should be noted in the readme next to kubernetes/origin

Reply:
The installer is a possible consumer of library-go. We would get a circular dependency. This having said: we only need the API type, right? So we have two choices:

1. move the install config API type to github.com/openshift/api.
2. handle the install config as pure JSON here, i.e. without dependency on the Golang types from the installer repo.